src/index.js

```diff
@@ -280,7 +291,8 @@ const cornerstoneTools = {
   SaveAs,
   enableLogger,
   disableLogger,
-  import: imp,
+  importInternalModule,
+  import: importInternalModule,
```

Comment:
I think not, since the whole reason we changed it is because the import keyword caused problems on some platforms iirc. I think we should hard-fork it here once and for all imo.